### PR TITLE
Fix cli.py

### PR DESCRIPTION
**Pull request by eggplants**
- print error message to stderr
- specify defaults of url

### CODE REVIEW

File: waybackpy/cli.py [hunk 1]
```diff
@@ -218,41 +218,41 @@ def save_urls_on_file(url_gen: Generator[str, None, None]) -> None:
...
 ) -> None:
     """\b
                          _                _
                         | |              | |
     __      ____ _ _   _| |__   __ _  ___| | ___ __  _   _
-    \ \ /\ / / _` | | | | '_ \ / _` |/ __| |/ / '_ \| | | |
-     \ V  V / (_| | |_| | |_) | (_| | (__|   <| |_) | |_| |
-      \_/\_/ \__,_|\__, |_.__/ \__,_|\___|_|\_\ .__/ \__, |
+    \\ \\ /\\ / / _` | | | | '_ \\ / _` |/ __| |/ / '_ \\| | | |
+     \\ V  V / (_| | |_| | |_) | (_| | (__|   <| |_) | |_| |
```
Comment (akamhy) [inline waybackpy/cli.py:254]: The ASCII art appears broken.

Correct art is:

```
                         _                _
                        | |              | |
    __      ____ _ _   _| |__   __ _  ___| | ___ __  _   _
    \ \ /\ / / _` | | | | '_ \ / _` |/ __| |/ / '_ \| | | |
     \ V  V / (_| | |_| | |_) | (_| | (__|   <| |_) | |_| |
      \_/\_/ \__,_|\__, |_.__/ \__,_|\___|_|\_\ .__/ \__, |
                    __/ |                     | |     __/ |
                   |___/                      |_|    |___/

```

**Review by akamhy — APPROVED** [review]
Everything else other than the ASCII art appears okay to me.

File: waybackpy/cli.py (hunk 1 above)
Comment (akamhy) [reply waybackpy/cli.py:254]: Why https://github.com/akamhy/waybackpy/pull/135/commits/d2a3946425ba550a79ad22bae96954024dd8fd2a?

Comment (eggplants) [reply waybackpy/cli.py:254]: Pylint says `Invalid escape sequence '\ ' (W605)`...

Comment (eggplants) [reply waybackpy/cli.py:254]: banner in `waybackpy --help` is no problem.

Comment (akamhy) [reply waybackpy/cli.py:254]: > banner in `waybackpy --help` is no problem.

Okay if `waybackpy --help` outputs it fine then merge it.

**Pull request merged**